code/src/GridAPI/Sort.ts

```diff
@@ -8,41 +8,95 @@ namespace GridAPI.Sort {
      *
      * @export
      * @param {string} gridID ID of the Grid that is to be to check from results.
-     * @returns {*}  {void}
+     * @returns {*}  {string} Return Message containing the resulting code from sorting columns and the error message in case of failure
      */
-    export function Clear(gridID: string): void {
+    export function Clear(gridID: string): string {
         PerformanceAPI.SetMark('Sort.Clear');
 
-        if (!OSFramework.Helper.IsGridReady(gridID)) return;
-        const grid = GridManager.GetGridById(gridID);
+        let returnMessage = {
+            isSuccess: true,
+            message: 'Success',
+            code: OSFramework.Enum.ErrorCodes.GRID_SUCCESS
+        };
```

Comment:
Please have an action, to be used cross API to do the set of this default object.

Reply:
I'm following the same names that Gabriel is using for consistency. I believe we need to review all these changes on a separate task since there are more PRs following this approach.